cmd/asset-syncer/sync.go

```diff
@@ -58,6 +61,15 @@ var syncCmd = &cobra.Command{
 		}
 
 		authorizationHeader := os.Getenv("AUTHORIZATION_HEADER")
+		// The auth header may be a dockerconfig that we need to parse
+		if os.Getenv("DOCKER_CONFIG_JSON") != "" {
```

Comment:
So if both are defined, we're using the docker config? Fine, just wanting to be sure it's intentional (the reason I'd suggested the opposite preference, to use the auth header if it's defined and only check the docker config json if auth header is not defined, was just because there's no error involved... ie. you can't have a situation where the auth header is defined and docker config is defined incorrectly so an error is raised).